src/webview/git-import/gitImportLoader.ts

```diff
@@ -275,16 +301,12 @@ function validateDevFilePath(event: any) {
     });
 }
 
-function showError(event: any, location: string, message: string): void {
+function showError(event: any): void {
     panel.webview.postMessage({
         action: event.action,
         status: false
     });
-    if (message.indexOf('already exists') !== -1) {
-        vscode.window.showErrorMessage(`Folder already exists on the selected ${location.substring(0, location.lastIndexOf('\\'))}`);
-    } else {
-        vscode.window.showErrorMessage('Error occurred while cloning the repository. Please try again.');
-    }
+    vscode.window.showErrorMessage('Error occurred while cloning the repository. Please click on Analyze Button and Try again.');
```

Comment:
'An error occurred while cloning the repository. Please click 'Analyze' button and try again'